resources/views/components/game/top-achievers/score-row.blade.php

```diff
@@ -0,0 +1,19 @@
+@props([
+    'rank' => 1,
+    'user' => null, // User
+    'score' => 0,
+])
+
+@if ($user->id === request()->user()->id)
+    <tr style='outline: thin solid'>
+@else
+    <tr>
+@endif
+
+        <td class="text-right">{{ $rank }}</td>
+        <td class="whitespace-nowrap">
+            <x-user.avatar display="icon" icon-size="xs" :user="$user" />
+            <x-user.avatar hasHref="true" :user="$user" />
+        </td>
+        <td class="text-right">{{ $score }}</td>
```

Comment:
We should probably use `localized_number()` on this value.

Reply:
done